### PR TITLE
[OC-1399] Notification Screen - Display the states in alphabetical order

### PR DESCRIPTION
**Pull request by vlo-rte**
Release notes : 

In Tasks section : 

[[OC-1399](https://opfab.atlassian.net/browse/OC-1399)] Notification Screen - Display the states in alphabetical order

### CODE REVIEW

File: ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts [hunk 1]
```diff
@@ -116,21 +134,33 @@ export class FeedconfigurationComponent implements OnInit {
...
                 }
-                this.processesStatesLabels.set(process.id, {processI18n: processI18n,
+                statesArray.sort((obj1, obj2) => {
+                    if (obj1.stateLabel > obj2.stateLabel) {
+                        return 1;
+                    }
+                    if (obj1.stateLabel < obj2.stateLabel) {
+                        return -1;
+                    }
+                    return 0;
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts:161]: Avoid duplication : Create a method sortByAlphabeticalOrder and send this method as parameter for statesArray.sort

File: ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts [hunk 2]
```diff
@@ -145,7 +175,26 @@ export class FeedconfigurationComponent implements OnInit {
...
+            });
+
+            this.processGroupsData.sort((obj1, obj2) => {
+                if (obj1.groupLabel > obj2.groupLabel) {
+                    return 1;
+                }
+                if (obj1.groupLabel < obj2.groupLabel) {
+                    return -1;
+                }
+                return 0;
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts:195]: See previous comment

File: ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts [hunk 3]
```diff
@@ -84,8 +87,23 @@ export class FeedconfigurationComponent implements OnInit {
...
+            }
+        });
+        this.processesWithoutGroup.sort((obj1, obj2) => {
+            if (obj1.processLabel > obj2.processLabel) {
+                return 1;
+            }
+            if (obj1.processLabel < obj2.processLabel) {
+                return -1;
+            }
+            return 0;
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts:106]: Avoid duplication : Create a method sortByAlphabeticalOrder and send this method as parameter for processesWithoutGroup.sort

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts (hunk 2 above)
Comment (vlo-rte) [reply ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts]: Done.

File: ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts (hunk 3 above)
Comment (vlo-rte) [reply ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts]: Done.

File: ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts (hunk 1 above)
Comment (vlo-rte) [reply ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts]: Done.

**Pull request merged**